src/main/java/ui/fileexplorer/FileChooser.java

```diff
@@ -0,0 +1,136 @@
+/* Licensed under MIT 2024. */
+package ui.fileexplorer;
+
+import ui.UserInterface;
+import ui.json.JSONHandler;
+
+import javax.swing.*;
+import javax.swing.filechooser.FileNameExtensionFilter;
+import java.io.File;
+
+public final class FileChooser {
+
+	private FileChooser() {
+		// Don't allow instances of this class
+	}
+
+	// Timesheet Format: Nachname_Vorname_Monat Jahr.pdf
+	private static final String FORMAT = "%s_%s 20%s";
+
+	public static String getDefaultFileName(UserInterface parentUI) {
+		return FORMAT.formatted(JSONHandler.getGlobalSettings().getNameUnderscoreFormat(), getGermanMonth(parentUI.getCurrentMonthNumber()),
+				parentUI.getYear());
+	}
+
+	public static File chooseFile(UserInterface parentUI, String title, FileChooserType chooserType) {
+		// Later: Implement Windows Version ?
+		// -> Later, I tried doing it simple and failed, and I don't have the time
+		// (currently) to figure it out
+		// Once I do, I'll make another pull request. It might not be as pretty, but
+		// because it remembers the
+		// folders, you don't need to go through the navigation hassle each time, so I
+		// think it's a fair
+		// compromise for now.
+		// - JustOneDeveloper
+		return chooseFileSwing(parentUI, title, chooserType);
```

Comment:
Could you explain why the current code does not work on windows?

Reply:
It does work on Windows, but the File chooser is the "shabby" one from Java, not the modern one we know from many Applications and what seems to be the Windows standard.

My goal was to implement the modern Windows Filechooser, but because there seems to be no simple way to do that in Java, I decided to leave it alone but leave the methods in a way where it can easily be implemented later.